ods_ci/tasks/Resources/Provisioning/Hive/OSP/create_fips.sh

```diff
@@ -8,13 +8,15 @@ export BASE_DOMAIN=${2:-$BASE_DOMAIN}
 export OSP_NETWORK=${3:-$OSP_NETWORK}
 export OSP_CLOUD=${4:-openstack}
 export OUTPUT_DIR=${5:-.${CLUSTER_NAME}_conf}
+export AWS_ACCESS_KEY_ID=${6:-$AWS_ACCESS_KEY_ID}
+export AWS_SECRET_ACCESS_KEY=${7:-$AWS_SECRET_ACCESS_KEY}
```

Comment:
Why do you need to pass credentials into the file, it is not too secured.
Can you set the AWS access before calling the script ?

Reply:
because we use the AWS hive credentials all over the provision process 
here inside the script, the export limited the effect of changing of the AWS keys to this script only without the necessary of switch between profiles

Reply:
as you can see from the console the key and secret masked